src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -27,17 +29,19 @@ With just a few additions your existing Dockerfile can be used with our Node.js
    "newrelic": "latest",
    ```
 
-   Install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
+You can update the value `latest` to install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
+
+2. Instrumentation is most reliable if the New Relic agent is required before your application starts. Your container setup may allow you to edit the `ENTRYPOINT` to include `newrelic` module first with the Node.js `-r`/`--require` flag when the node command is invoked. If your Dockerfile contains any of these start commands, you can change them this way:
```

Comment:
It's best to lead procedures with a clear, direct action that needs to happen. If conceptual info is necessary to help people understand how to do it or why they're doing it, we should add that after the fact. This allows our power users to get to the meat of the procedure while prociding context for newer/less experienced users (that power users can skim past) For example, in step 1, we clearly tell them to "Add X to Y". 

What is the action in its most distilled form here? Some suggestions:

* Add the Node.js ``-r`/`--require` flag to the `ENTRYPOINT` so the `newrelic` module appears first. Instrumentation is most reliable if the New Relic agent is required before your app starts. If your container setup doesn't allow this action, then try some of these alternatives:
    * something
    * something
    * something

Reply:
I have a proposed change I'll push in the next commit.